packages/theme/src/index.ts

```diff
@@ -159,12 +160,29 @@ const extension: JupyterFrontEndPlugin<ILauncher> = {
       }
     });
 
+    commands.addCommand(CommandIDs.releases, {
+      label: "What's new in latest",
```

Comment:
We should specify a human-readable tool tip like we do for other tiles

![image](https://user-images.githubusercontent.com/13068832/184071501-97452652-74cb-46ac-9935-bd7defcace8f.png)


e.g.

```suggestion
      label: "What's new in latest",
      caption: "What's new in this release",
```